server/channels/app/plugin.go

```diff
@@ -378,6 +378,25 @@ func (ch *Channels) ShutDownPlugins() {
 	}
 }
 
+func (a *App) getPluginManifests() ([]*model.Manifest, error) {
+	pluginsEnvironment := a.GetPluginsEnvironment()
+	if pluginsEnvironment == nil {
+		return nil, model.NewAppError("GetPluginManifests", "app.plugin.disabled.app_error", nil, "", http.StatusNotImplemented)
```

Comment:
I'm not 100% convinced this is the correct HTTP status code. What was the reasoning so I can understand?

Reply:
Consistency. The other plugin-related methods return the same status code, e.g. in https://github.com/mattermost/mattermost/blob/d5cc2eb2f607c592620e4f91a9fc7435033f8e78/server/channels/app/plugin_install.go#L429-L432.

Reply:
Consistency. The other plugin-related methods return the same status code, e.g. in https://github.com/mattermost/mattermost/blob/d5cc2eb2f607c592620e4f91a9fc7435033f8e78/server/channels/app/plugin_install.go#L429-L432.

Reply:
Consistency. The other plugin-related methods return the same status code, e.g. in https://github.com/mattermost/mattermost/blob/d5cc2eb2f607c592620e4f91a9fc7435033f8e78/server/channels/app/plugin_install.go#L429-L432.